runtime/lua/vim/lsp.lua

```diff
@@ -273,8 +273,12 @@ end
 --@returns (function) Memoized function
 local function once(fn)
```

Comment:
Why is this change necessary?

Reply:
If a function is just used for side-effects `once` will not run it once

Reply:
Ah, I guess that's fine then.